pysphero/device_api/power.py

```diff
@@ -10,6 +10,15 @@ class BatteryVoltageStates(UnknownEnumMixing, Enum):
     low = 0x02
     critical = 0x03
 
+class BatteryLMQStates(UnknownEnumMixing, Enum):
+    unknown = 0x0
```

Comment:
Extra line, because UnknownEnumMixing is used

Reply:
Ok will remove

Reply:
Done